kubernetes-python/requirements.txt

```diff
@@ -0,0 +1,2 @@
+pulumi>=0.16.14
+pulumi_kubernetes>=0.20.3
```

Comment:
```suggestion
pulumi-kubernetes>=0.20.3
```

Reply:
Should we make a similar change in our other Python templates? For example, the `aws-python` template uses `pulumi_aws`, should it be `pulumi-aws`?

https://github.com/pulumi/templates/blob/84252aa51b0e41af4c279b204833099c67636ac2/aws-python/requirements.txt#L2